pages/src/_data/buildingsettings.yml

```diff
@@ -40,4 +40,29 @@ glassblower:
       - name: Priority
         description: This is the default setting. The hut will try to use the recipe that is higher up in their recipe list first.
       - name: Warehouse Stock
-        description: The hut will look in the warehouse first to see what resource you have more of before deciding what recipe it will use.
\ No newline at end of file
+        description: The hut will look in the warehouse first to see what resource you have more of before deciding what recipe it will use.
+guardtower:
+  - name: Task
+    description: This is where you can choose if you want the Guard to patrol, follow, or guard.
+    options:
+      - name: Patrol
+        description: The guard will patrol an area you designate in <strong>Patrol Settings</strong>. 
+      - name: Guard
+        description: You can set one area for the Guard to stay in. When you click "Set Target", you will be taken to the <strong>Selection Tools</strong> tab, where you can get the the Guard Scepter and designate the guard location.
+      - name: Follow
+        description: The guard will follow you around as your personal bodyguard, protecting you or fighting alongside you. They will even go outside the colony when following! You can designate how they follow you in <strong>Follow Settings</strong>. 
+      - name: Patrol Mine
+        description: The Guard will patrol the level of a <a href="../../source/buildings/mine">Mine</a> the <a href="../../source/workers/miner">Miner</a> is currently mining at. You must also select the guard in the Mine GUI in order for the guard to being patrolling.
```

Comment:
The wording here is a little awkward at the end. I think you can also have guards patrol levels the miner isn't currently working. Maybe the following:

> description: The Guard will patrol a level of a <a href="../../source/buildings/mine">Mine</a>. You must also assign that level in the Mine GUI.

Reply:
I meant that it doesn't automatically patrol only the level the miner is currently mining, right?

Reply:
Sorry, I can't read this early in the morning. Now I see what you meant. In that case, this can be edited to reflect that and still be shorter than the current commit.